flake-module.nix

```diff
@@ -56,6 +56,13 @@ in
                   Port to serve process-compose's Swagger API on.
                 '';
               };
+              disable-tui = mkOption {
+                type = types.bool;
+                default = false;
```

Comment:
I think this should be more of:

```nix
tui = mkOption {
  type = types.nullOr types.bool;
  default = null;
};
```

1. `tui` is simpler and more straightforward name for the option. It is a noun, which is what we want for options in general, rather than a verb ("disable").
2. If the default is null, the CLI need not be modified at all. This way users can pass `-t=..` in the command line themselves if they choose to. 
    - We should probably change the type `port` as well in this manner.